src/utils/webrtc/simplewebrtc/getscreenmedia.js

```diff
@@ -23,7 +23,13 @@ const getUserMedia = function(constraints, callback) {
 // cache for constraints and callback
 const cache = {}
 
-module.exports = function(mode, constraints, cb) {
+/**
+ *
+ * @param mode
```

Comment:
Will be super to add some docs

Code that might be related:
https://github.com/HenrikJoreteg/webrtc.js/blob/f7da40fcc599e5650c210f7864b4ebeeefbbc31b/webrtc.bundle.js